frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -879,18 +879,33 @@ export const widgets = [
       textColor: {
         type: 'color',
         displayName: 'Text Color',
-        validation: { schema: { type: 'string' } },
+        validation: { 
+          schema: { type: 'string' }, 
+        },
       },
       borderRadius: {
         type: 'code',
         displayName: 'Border radius',
-        validation: { schema: { type: 'union', schemas: [{ type: 'string' }, { type: 'number' }] } },
+        validation: { 
```

Comment:
Please revert back to formatting changes.
And try to follow the repository's Lint rules.